test/dynamo/test_functions.py

```diff
@@ -972,16 +972,6 @@ def test_distributed_is_available(x):
         else:
             return x - 1
 
-    @unittest.skipIf(
-        not torch.distributed.is_available(), "requires distributed package"
-    )
-    @make_test
-    def test_distributed_is_initialized(x):
-        if torch.distributed.is_initialized():
-            return x + 1
-        else:
-            return x - 1
-
```

Comment:
This test never tested anything - it was 100% in skipfiles, no graph, no compiled code.